corehq/apps/sms/templates/sms/usage_report.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'hqwebapp/base_section.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load timezone_tags %}
+{% requirejs_main "sms/js/sms_usage_report" %}
+{% block page_content %}
+    <form method="post">
+        {% csrf_token %}
+        <div class="form-group">
+            <label for="date_range_selector">Select Date Range</label>
```

Comment:
`trans` here, for the button below, and for the placeholder. For the placeholder, use `trans_html_attr`, which takes care of escaping quote marks so the translation can be safely used as an html attribute.